src/sentry/conf/server.py

```diff
@@ -935,6 +937,7 @@ def SOCIAL_AUTH_DEFAULT_USERNAME() -> str:
     Queue("nudge.invite_missing_org_members", routing_key="invite_missing_org_members"),
     Queue("auto_resolve_issues", routing_key="auto_resolve_issues"),
     Queue("on_demand_metrics", routing_key="on_demand_metrics"),
+    Queue("spans.process_segment", routing_key="spans.process_segment"),
```

Comment:
iirc, segment is the new name for a transaction? Will this queue be for processing segments for just batches of spans? If it's just batches of spans, perhaps consider renaming it.

Reply:
segments, one segment id per task